src/app.js

```diff
@@ -1,14 +1,10 @@
 require('dotenv').config();
 const { Client } = require('discord.js');
 const config = require('./config.json');
-const { 
-  kickMember, 
-  BanUser, 
-  ClearMessages 
-} = require('./commands/general');
+const commands = require('./commands/general.js');
 
 // =========== Initialisations ===========
-const primaryPrefix = config.primaryPrefix;
+const primaryPrefix = config.primary_prefix;
```

Comment:
Incoming commit from cleanup branch fixes this issue in the config file. Potential little mix up after merging both commits. Easy fix.